Description: Escape minus signs to fix lintian warning
  I: fio: hyphen-used-as-minus-sign usr/share/man/man1/fio.1.gz:934
Author: Martin Steigerwald <ms@teamix.de>

---
Index: fio/fio.1
===================================================================
--- fio.orig/fio.1	2011-10-05 11:23:02.302972277 +0200
+++ fio/fio.1	2011-10-05 11:28:04.442172709 +0200
@@ -931,7 +931,7 @@
 Overwrite the default list of percentiles for completion
 latencies. Each number is a floating number in the range (0,100], and
 the maximum length of the list is 20. Use ':' to separate the
-numbers. For example, --percentile_list=99.5:99.9 will cause fio to
+numbers. For example, \-\-percentile_list=99.5:99.9 will cause fio to
 report the values of completion latency below which 99.5% and 99.9% of
 the observed latencies fell, respectively.
 .SH OUTPUT
